content-src/components/Base/Base.jsx

```diff
@@ -106,6 +106,25 @@ export class BaseContent extends React.PureComponent {
   constructor(props) {
     super(props);
     this.openPreferences = this.openPreferences.bind(this);
+    this.onWindowScroll = this.onWindowScroll.bind(this);
+    this.state = {fixedSearch: false};
+  }
+
+  componentDidMount() {
+    global.addEventListener("scroll", this.onWindowScroll);
```

Comment:
before I add the debounce... i sit needed or is that a premature optimization?

Reply:
nah definitely a good idea

Reply:
scroll is already janky on some machines